samplesheet.wdl

```diff
@@ -1,4 +1,5 @@
 version 1.0
+
 struct Readgroup {
```

Comment:
This complete file should not be in tasks. This can be pipeline specific

Reply:
@rhpvorderman placed it here, the idea being that this is a basic version that could easily be reused. If a pipeline needs something special, they don't have to use this one. I'm not too opinionated on whether this should be here or not, but maybe Ruben has something to say about this?

Reply:
Is this not why we have pipeline-template?